monoio/src/driver/op/read.rs

```diff
@@ -186,20 +186,20 @@ impl<T: IoVecBufMut> OpAble for ReadVec<T> {
 
     #[cfg(all(any(feature = "legacy", feature = "poll-io"), windows))]
     fn legacy_call(&mut self) -> io::Result<u32> {
-        let mut bytes_recved = 0;
+        let mut bytes_received = 0;
         let ret = unsafe {
             WSARecv(
                 self.fd.raw_socket() as _,
                 self.buf_vec.write_wsabuf_ptr(),
                 self.buf_vec.write_wsabuf_len().min(u32::MAX as usize) as _,
-                &mut bytes_recved,
-                std::ptr::null_mut(),
+                &mut bytes_received,
+                &mut 0,
```

Comment:
The lpFlags here is used as both in and out. So it has to be able to read and write.
Have you tried to implement it like rust std? In std impl, it declare 2 temp variable nread and flags, and then pass the mut references.

https://github.com/rust-lang/rust/blob/master/library/std/src/sys/pal/windows/net.rs#L253